<launch>

  <!-- Camera Image Transport to raw--> I
  <node name="republish" type="republish" pkg="image_transport" output="screen" args="compressed in:=/raspicam_node/image raw out:=/camera/rgb/image_raw" />

  <!-- <node name="pano_server" pkg="pano_ros" type="capture_server.py"/> -->

  <node name="turtlebot3_panorama" pkg="turtlebot3_panorama" type="panorama">
    <param name="default_mode" value="1"/>
    <param name="default_pano_angle" value="6.28318530718"/> <!-- 2 * Pi -->
    <param name="default_snap_interval" value="2.0"/>
    <param name="default_rotation_velocity" value="0.3"/>
    <param name="camera_name" value="camera/rgb"/>
    <param name="bag_location" value="/tmp/turtlebot3_panorama.bag"/>
    
  </node>
</launch>
